x/superfluid/keeper/concentrated_liquidity_test.go

```diff
@@ -232,7 +232,7 @@ func (s *KeeperTestSuite) TestAddToConcentratedLiquiditySuperfluidPosition() {
 	}
 }
 
-func (s *KeeperTestSuite) SetupSuperfluidConcentratedPosition(ctx sdk.Context, superfluidDelegated, superfluidUndelegating, unlocking bool, owner sdk.AccAddress) (positionId, lockId uint64, amount0, amount1 osmomath.Int, valAddr sdk.ValAddress, poolJoinAcc sdk.AccAddress) {
+func (s *KeeperTestSuite) SetupSuperfluidConcentratedPosition(ctx sdk.Context, superfluidDelegated, superfluidUndelegating, unlocking bool, owner sdk.AccAddress) (positionId, lockId uint64, amount0, amount1 osmomath.Int, valAddr sdk.ValAddress, poolJoinAcc sdk.AccAddress) { //nolint:revive // TODO: refactor this function
```

Comment:
Just curious, what's revive? Isn't it supposed to be a replacement of golint? 👀

Reply:
Revive is a lot more than a replacement of golint, but it also does that too :)

here's their web page:

* https://revive.run/

Also, I we configured it pretty intensely for ibc-go recently:

https://github.com/cosmos/ibc-go/blob/2007367d74a9366fbf1ef7f3b324aaa2fbf520cf/.golangci.yml#L46-L114

It can be run as a part of golangci-lint or on its own.